src/main/java/backend/UpdateController.java

```diff
@@ -144,4 +145,22 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
         // No sort or updated, return sort by descending ID, which is the default.
         return Qualifier.getSortComparator(models, "id", true, false);
     }
+
+    /**
+     * Applies the count qualifier to the set of issues obtained after the sorting order is applied.
+     *
+     * @param issueList Issue list obtained after filtering and sorting.
+     * @param filterExpr The filter expression of the particular panel.
+     * @return The maximum count of issues
+     */
+    private int determineMaxCount(List<TurboIssue> issueList, FilterExpression filterExpr){
+        for (Qualifier metaQualifier : filterExpr.find(Qualifier::isMetaQualifier)) {
+            if (metaQualifier.getType() == QualifierType.COUNT) {
+                int countQualifierNum = metaQualifier.getNumber().get();
```

Comment:
This would fail for an input like `count:a`, because then `getNumber()` would be empty. You should probably check for the presence of `getNumber()` in the condition. Unfortunately we don't have a way to statically rule out invalid inputs yet.